<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default2" uuid="64dba02b-313d-4459-a798-7d1a4f625508">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/untitled3/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:E:\web_jy\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="MySQL for 5.1 - web_jy@39.108.139.175" uuid="dc5a2e31-cb18-433c-a475-d8d984d83f7b">
      <driver-ref>mysql</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>com.mysql.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://39.108.139.175:3306/web_jy</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="f9445d01-5236-431c-a2ca-2ebf8c6b44d8">
      <driver-ref>mysql.8</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/untitled3/settings.py</remarks>
      <jdbc-driver>com.mysql.cj.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://39.108.139.175:3306/web_jy</jdbc-url>
    </data-source>
  </component>
</project>